src/initializers/face-detection.js

```diff
@@ -0,0 +1,31 @@
+// const ReactGA = require('react-ga');
```

Comment:
Repurpose this.  It isn't used at all right now...just an abandoned idea.  This could be turned into the connection to how Kegbot responds to seeing a face.  Basically, it could create a kegbot, subscribe it to the store, and set it on a loop to think (or have the store updates drive this -- don't think I like this idea).